### PR TITLE
allow changing minecraft data dir + allow opening firewall

### PR DESCRIPTION
**Pull request by cwoac**
Two minor tweaks: allow overriding the data directory + (if chosen) open the firewall.

### CODE REVIEW

File: nixos/modules/services/games/minecraft-server.nix [hunk 1]
```diff
@@ -15,7 +16,23 @@ in
...
+        type = types.path;
+        default = "/var/lib/minecraft";
+        description = ''
+          Directory to store minecraft database and other state/data files.
+        '';
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = true;
```
Comment (domenkozar) [inline nixos/modules/services/games/minecraft-server.nix]: The consensus is not to enable ports by default. You can add documentation to `enable` option what is needed for service to function correctly (and provide an example)

**Comment by cwoac** [comment]
Gah. Could have sworn I'd set the firewall option to `false` by default. Fixed.

**Comment by jagajaga** [comment]
Squash commits please.

**Comment by cwoac** [comment]
will fix. requires a fresh req.

**Comment by jagajaga** [comment]
You can squash via rebase and make force push.. That will help not to create additional PR.

**Pull request closed without merging**